static/templyte/src/app/upload-success-dialog/upload-success-dialog.component.html

```diff
@@ -1,3 +1,3 @@
 <h2>Upload Successful</h2>
-<p>Congrats! You're on your way to template glory!</p>
-<button mat-button mat-dialog-close>Close</button>
\ No newline at end of file
+<p>Your template has been uploaded successfully!</p>
```

Comment:
You don't like my success message? It's so quirky and fun, yours is boring! We should change it back.